pkg/telemetry/telemetry.go

```diff
@@ -0,0 +1,106 @@
+package telemetry
+
+import (
+	"crypto/sha1"
+	"fmt"
+	"os"
+	"time"
+
+	"net/http"
+	"net/url"
+
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+)
+
+// eventType
+const eventType = "library-os-ng"
+
+// actionName
+const actionName = "Piper Library OS"
+
+// LibraryRepository that is passed into with -ldflags
+var LibraryRepository string
+
+// SiteID ...
```

Comment:
What is the semantics of a site id? (just curious)

Reply:
Unique ID where the SWA data is assigned to.